schemas/resource-schema.json

```diff
@@ -191,6 +196,20 @@
               "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$"
             }
           }
+        },
+        "_dd": {
+          "type": "object",
+          "description": "Internal properties",
+          "properties": {
+            "span_id": {
+              "type": "integer",
+              "description": "span identifier in decimal format"
+            },
+            "trace_id": {
+              "type": "integer",
```

Comment:
Is the plan to keep [63 bits integers as trace id](https://github.com/DataDog/browser-sdk/pull/461/files#diff-ad28acad0be8055df13870436c79de04R82-R105) ? If it is, I think the type should be a string, as it may be difficult to represent 63 bits integers values in JS/json. (same for span_id)